src/sentry/static/sentry/app/components/scoreBar.jsx

```diff
@@ -63,6 +63,7 @@ const StyledScoreBar = styled(ScoreBar)`
     flex-direction: column-reverse;
     justify-content: flex-end;
   `};
+  ${p => !p.vertical && 'min-width: 80px;'}
```

Comment:
is it easier to read if the above is a ternary?

Reply:
Fair. I can combine them.